From: =?utf-8?q?Alexis_Bienven=C3=BCe?= <pado@passoire.fr>
Date: Fri, 27 Sep 2024 23:19:10 +0200
Subject: Strip build date from version_report_string

Strip the build date from the version_report_string return value,
for reproducibility reasons.
---
 gwlib/utils.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/gwlib/utils.c b/gwlib/utils.c
index f9e82b3..4ad6d9e 100644
--- a/gwlib/utils.c
+++ b/gwlib/utils.c
@@ -551,7 +551,7 @@ Octstr *version_report_string(const char *boxname)
     struct utsname u;
 
     uname(&u);
-    return octstr_format(GW_NAME " %s version `%s'.\nBuild `%s', compiler `%s'.\n"
+    return octstr_format(GW_NAME " %s version `%s'.\nCompiler `%s'.\n"
                          "System %s, release %s, version %s, machine %s.\n"
              "Hostname %s, IP %s.\n"
              "Libxml version %s.\n"
@@ -587,10 +587,9 @@ Octstr *version_report_string(const char *boxname)
              "Using %s malloc.\n",
              boxname, GW_VERSION,
 #ifdef __GNUC__ 
-             (__DATE__ " " __TIME__) ,
              __VERSION__,
 #else 
-             "unknown" , "unknown",
+             "unknown",
 #endif 
              u.sysname, u.release, u.version, u.machine,
              octstr_get_cstr(get_official_name()),
